types/cbor-js/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for cbor-js 0.1.0
+// Project: https://github.com/paroga/cbor-js
+
+export function encode(data: any): ArrayBuffer;
+
+export function decode(encoded: ArrayBuffer): any;
```

Comment:
```suggestion
export function decode(encoded: ArrayBuffer): any;

export as namespace CBOR;
```
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
https://github.com/paroga/cbor-js/blob/master/cbor.js#L399 (and others)